src/Components/Publishing/Sections/ArtworkCaption.tsx

```diff
@@ -23,13 +23,16 @@ interface StyledArtworkCaptionProps extends React.HTMLProps<HTMLDivElement> {
 }
 
 export class ArtworkCaption extends React.Component<ArtworkCaptionProps, null> {
-  joinParts(children, delimiter = ", ") {
+  joinParts(children, key, delimiter = ", ") {
```

Comment:
Interesting note... I actually wrote a [Join component](https://github.com/artsy/reaction/blob/bbb42c55437f61187967c9c5baefce3ebc79f3bd/src/Styleguide/Elements/Join.tsx#L7). I wonder if it could be used here. Maybe that's a separate PR though.

Reply:
Nice! But yeah that should probably be done separately and maybe somebody from the Grow team instead. /cc @eessex